rich-default/Dockerfile

```diff
@@ -233,37 +141,6 @@ RUN apt-get update && \
     apt-get clean
 ENV PATH=$PATH:/usr/lib/rstudio-server/bin
 
-# Installs the jupyter-rsession-proxy extension
-RUN conda install -yq -c conda-forge jupyter-rsession-proxy && \
-    conda clean -tipsy
-
-# Install pymol
-#RUN conda install -yq -c schrodinger pymol
-
-# Install ipymol
-# Installs from a git repo because pymol fetch fails when installing the pip package
-RUN pip install git+https://github.com/cxhernandez/ipymol.git@2a30d6ec1588434e6f0f72a1d572444f89ff535b
-
-RUN pip install pypdb
-RUN pip install biopandas
```

Comment:
These both are on conda forge I think. Use conda.

Reply:
The current environment uses pypdb and biopandas that are newer versions than are currently used on conda. I can try downgrading them though and seeing if they will work.